src/sound/sound_handler.h

```diff
@@ -269,6 +271,7 @@ class SoundHandler {
 	 * called. false = can be played
 	 */
 	static bool backend_is_disabled_;
+	bool use_custom_songset_instead_ingame;
```

Comment:
Use trailing underscore for member variables
```suggestion
	bool use_custom_songset_instead_ingame_;
```